Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc

```diff
@@ -230,15 +249,14 @@ void DDHGCalEEAlgo::constructLayers(const DDLogicalPart& module,
 #endif
       } else {
 	DDSolid solid = DDSolidFactory::tubs(DDName(name, nameSpace_), 
-					     hthick, rinB, routF, 0.0,
-					     CLHEP::twopi);
+					     hthick, rinB, routF, 0.0, 2._pi);
 	glog = DDLogicalPart(solid.ddname(), matter, solid);
 #ifdef EDM_ML_DEBUG
 	edm::LogVerbatim("HGCalGeom") << "DDHGCalEEAlgo: " << solid.name() 
 				      << " Tubs made of " << matName << ":"
 				      << &matter << " of dimensions " << rinB 
 				      << ", " << routF << ", " << hthick
-				      << ", 0.0, " << CLHEP::twopi/CLHEP::deg
+				      << ", 0.0, " << convertRadToDeg(2._pi)
```

Comment:
If the `2._pi` here is a value that could change in the future, then it needs to be replaced with a constant. If the `2._pi` will never change in future versions of the code, then simply write `"360.0"`. What purpose is served by obfuscating "360.0"?

Reply:
done